README.md

```diff
@@ -1,8 +1,60 @@
 ## What's this?
 
-This repo holds metadata about the Flutter framework. The metadata is largely
-useful for tooling (the Flutter IntelliJ plugin, the VS Code Dart plugin, ...).
+This repo holds generated metadata about the Flutter framework.
 
-<!-- TODO(devoncarew): Have a describing the various metadata provided -->
+The metadata is useful for Flutter related tooling and is currently used by
+Flutter IDE tools such as the Flutter IntelliJ and VS Code plugins.
 
-<!-- TODO(devoncarew): Have a section listing tools that use this data -->
+If you're using this metadata, please let us know! You can open an issue on the
+repo or add your tool to this readme via a pull request.
+
+## Tools that use this metadata
+
+- the [Flutter IntelliJ] plugin (for IntelliJ and Android Studio)
+- the [Dart-Code] VS Code plugin
+
+## Available metadata
+
+The generated framework data lives at
+https://github.com/flutter/tools_metadata/tree/master/resources.
+
+### Widget Catalog
+
+Available
+[here](https://github.com/flutter/tools_metadata/blob/master/resources/catalog/widgets.json),
+this is the full set of all public widgets from the Flutter framework. It
+includes information like the widget name, it's parent, the library it's defined
+in - like `material`, and the description of the widget.
+
+### Colors
+
+This is a mapping of Flutter framework color names to color values, for both the
+Material and Cupertino sets of colors. It's available in property file and json
+formats.
+
+https://github.com/flutter/tools_metadata/tree/master/resources/colors
+
+### Icons
+
+This is a mapping of:
+
+- font codepoints to icon names, and
+- icon names to file paths for preview icons
+
+This information is available for both the Material and Cupertino icons. In
+addition, each framework icon has a 16x16 png preview image.
```

Comment:
@DanTup - I did check, and it looks like we're generating 16x16 icon previews. From memory, I'd thought we were generating 32x32 sized previews.

I'll confirm what size works best for IntelliJ. I _think_ it would use 32x32 sized icons, but display them as 16x16 on higher resolution monitors. I'd want to verify that before we change the size, but, does VS Code have a size preference?

Reply:
We seem to generate both - the 32x32 ones have `@2x` in the filename?

VS Code is using the `@2x` ones and setting the size to 32x32 though, so that's probably not actually seeing the high-res benefit (it's 32 "logical" pixels, so we really want a 64x64 `@2x` version).

So if your plan is to bump them up to generate 32x32 standard icons, and 64x64 `@2x` variants, that sounds good to me!

Reply:
Ah! I don't know why, but I'd missed the `@2x` ones. IntelliJ is using them as 16x16 logical icons.

I don't know how I feel about generating 3 preview images for each icon; probably something to think about.

Reply:
I checked again, and I might've been wrong about VS Code - I think they're being rendered at 16x16 (the macOS screenshot tool seems to use actual pixels when dragged out on the high def screen, but I assumed it was logical pixels).

So I think what we have (32x32 @2x images to render at 16x16) is actually good for VS Code (and sounds like the same as IntelliJ is doing).